pasta_eln/GUI/project.py

```diff
@@ -213,20 +210,20 @@ def executeAction(self) -> None:
         #update sidebar, show projects
         self.comm.changeSidebar.emit('redraw')
         self.comm.changeTable.emit('x0','')
-    elif menuName == 'scanProject':
+    elif identifier[0] is Command.SCAN:
       self.comm.backend.scanProject(self.comm.progressBar, self.projID, self.docProj['-branch'][0]['path'])
       self.comm.changeSidebar.emit('redraw')
       showMessage(self, 'Information','Scanning finished')
-    elif menuName == 'projReduceWidth':
+    elif identifier[0] is Command.REDUCE:
```

Comment:
Would it not be better to be specific enum names? REDUCE what? As we had it before projReduceWidth is more reaable, similarly can we have more specific names for the commands?

Reply:
So: we should use more elaborate enum names: like PROJECT_REDUCE_WIDTH?

Reply:
Yes, this would be more readable then right? Otherwise document the values, either way should be Okay